src/java/com/google/cloud/bigquery/dwhassessment/hooks/logger/EventRecordConstructor.java

```diff
@@ -113,6 +114,7 @@ private GenericRecord getPreHookEvent(HookContext hookContext) {
         .set("OperationId", hookContext.getOperationId())
         .set("DatabasesRead", getDatabasesFromEntitySet(plan.getInputs()))
         .set("DatabasesWritten", getDatabasesFromEntitySet(plan.getOutputs()))
+        .set("Database", SessionState.get().getCurrentDatabase())
```

Comment:
Optional: to make the code on the service side a bit simpler, you could name the field "DefaultDatabase" – then we won't need to remap it.

Reply:
done

Reply:
MigrationAssessmentLoggingHookTest.java complains that on TestUtils.java:148 it got 0 instead of expected 1. I dont know root cause but adding SessionState fixes it

Reply:
What I mean is that there is no test where `DefaultDatabase` is filled. (i.e. it is implicitly null). Would be nice to have a test where the output record contains non-null `DefaultDatabase` column.

Reply:
created new pull request